.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,23 @@
+name: Python-pre-releases
+
+on:
+  pull_request:
+  schedule:
```

Comment:
FYI, likely doesn't need to run on each PR, but if this isn't very expensive, it may make sense as our other tests take longer.